lib/src/components/NxIconDropdown/NxIconDropdown.tsx

```diff
@@ -47,7 +47,7 @@ const NxIconDropdown: FunctionComponent<Props> = function NxIconDropdown(props)
               aria-haspopup="true"
               aria-expanded={isOpen}
               title={title}>
-      <NxFontAwesomeIcon icon={icon || faEllipsisV}/>
+      <NxFontAwesomeIcon className="nx-icon--colorful" icon={icon || faEllipsisV}/>
```

Comment:
This has no effect

Reply:
> This has no effect

Removed here: 542e7e67935e50ab1f2f4d7efd8e4971846e2ad3